tensorflow/stream_executor/cuda/cuda_driver.cc

```diff
@@ -584,7 +587,8 @@ bool DeviceOptionsToContextFlags(DeviceOptions device_options, int *flags) {
     // TODO(leary) Need to see if NVIDIA can expunge the leakiness in their
     // context creation: see http://b/13248943
 
-    res = dynload::cuCtxCreate_v2(&new_context, flags, device);
+    res = dynload::cuDevicePrimaryCtxSetFlags(device, flags);
```

Comment:
These functions were introduced in 7.0. I will add checks for earlier versions of CUDA.